### ISSUE TITLE
Cannot create property 'doc' on string 'memory'

### ISSUE THREAD

**Issue by edsu** [nextapps-de/flexsearch#120]
I'm following this example from the README.md using v0.6.30:

```javascript
var index = new FlexSearch({
    doc: {
        id: "id",
        field: {
            title: {
                encode: "extra",
                tokenize: "reverse",
                threshold: 7
            },
            cat: {
                encode: false,
                tokenize: function(val){
                    return [val];
                }
            },
            content: "memory"
        }
    }
});
```

and I get this error:

```
TypeError: Cannot create property 'doc' on string 'memory'
    at w.init (/Users/edsu/Projects/airwaves/node_modules/FlexSearch/dist/flexsearch.node.js:20:153)
    at new w (/Users/edsu/Projects/airwaves/node_modules/FlexSearch/dist/flexsearch.node.js:8:263)
```

It would be nice to be able to use the "memory" shorthand for those settings. Is this actually possible and the documentation has it wrong?

**Comment by ts-thomas** [comment]
Sorry there was an issue with the version. The package.json was updated but the new code from 0.6.30 never gets published, because it was not stable. Will be fixed on 0.7.0 which is actually available as preview.

**Comment by ts-thomas** [comment]
Please change over to the version >= 0.7.x, thanks a lot.
